platform/ios/CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
-## 4.4.0
+## master
+* Added an `MGLSymbolStyleLayer.symbolZOrder` property for forcing point features in a symbol layer to be layered in the same order that they are specified in the layer’s associated source. [#12783](https://github.com/mapbox/mapbox-gl-native/pull/12783)
```

Comment:
This change only occurs on master and should not be cherry-picked. Use `git cherry-pick -e` to edit before committing the cherry-pick.